### ISSUE TITLE
Scalloping of surface of live and spleen

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10109]
**Preferred term label:**
Scalloping of surface of live and spleen

**Synonyms**

**Definition (free text, please give PubMed ID)**
caused by loculated accumulations of mucin

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pseudomyxoma Peritonei

**Your nano-attribution (ORCID)**
